### ISSUE TITLE
20. Restoring plr

### ISSUE THREAD

**Issue by marcokitt2000** [vasyl83/SV07update#2]
When everything is done can we restore the plr function. In printer.cfg i have ; 
Every text  what you say. Can we set it back?

Thnx for this great work.

Gr. Marco you can mail me marcokitt2000@hotmail.com

I have don it bud when i use usb stick where img file on it it doest image good to emmc, what i have done i have adapter emmc from makerbase. i have use image tool to emmc and start it than go further. It works all so plr all so Christian Vick kpa 3.0 b91.

If there is a way to get back sovol startup screen thats be nice. When i start i see begin txt horizontal than solar screen than is ok. but thats what i like from sovol nothing to see. but there is a strange thing wifi (realtek chip has weak signal on old sovol os) signal is little bit better. i have version hw 1.0 without fan connector for cooling klipad. hw 1.1 has it and look like the same. hw 1.2 is other lcd cables are now connected under lcd and fan connection. but wifi is not realtek other branch but better for use.

Allso when i reboot step 3 hold space bar on pc 10 sec pwr button klipad than it shuts down i must press 1x time pwr button than goes further.

And the beeber.... sh file cant created it says macros map not ther. i do it in mainsail make a map than it works. only the beeber doesn hear it all so when press buttons on klipscreen i know it can be off. but it will come to work now test printer.

Keep the good work i hope sovol will add improved id.

**Comment by marcokitt2000** [comment]
test all things are good.

question can i install the sovol klipper screen back or from old version etc?

but i like both only sovol has where here #pinned for led on or of 0 to 100% led icoon.

**Comment by vasyl83** [comment]
yes simply use the sovol .img instead of bookworm. dd it to the emmc and reboot.

**Comment by marcokitt2000** [comment]
I mean use the klipperscreen data from sovol img to use for bookwurm.

But i still testing... i got 32gb orginal emmc v1 ftom mks makerbase.
Extra free memory for gcode.

Thxxxxx for replay.


Marco

**Comment by marcokitt2000** [comment]
I hope someone here can tell me to use sovol klipperscreen(sovol adjustments) instead from klipperscreen. Or how make my own klipperscreen.

**Comment by vasyl83** [comment]
If you want to customize klipperscreen (as sovol changes are simple customizations) you'll have better luck on klipperscreen github.
